src/components/facets/facet.component.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import 'chance';
+import { FacetsModule } from './facets.module';
+import { Facet } from './models/facet';
+
+@Component({
+    selector: 'app',
+    template: `
+    <div class="row">
+        <div class="col-md-12">
+            <ux-facet-container>
+                <ux-facet-check-list header="Authors" [facets]="facets"></ux-facet-check-list>
+            </ux-facet-container>
+        </div>
+    </div>
+    `
+})
+export class FacetCheckListTestComponent {
+
+    facets: Facet[] = [];
+
+    constructor() {
+
+        // generate some facets
+        for (let idx = 0; idx < 30; idx++) {
```

Comment:
No need for 30 items in a unit test. I suggest 3 with names and values that are not randomly generated.